packages/frontend/package.json

```diff
@@ -13,11 +13,12 @@
     "@aws-sdk/s3-request-presigner": "3.627.0",
     "@aws-sdk/util-create-request": "3.622.0",
     "@aws-sdk/util-format-url": "3.609.0",
-    "@reach/router": "1.3.4",
+    "@reach/router": "^1.3.4",
     "p-event": "^6.0.1",
-    "react": "18.2.0",
+    "react": "^19.0.0",
     "react-bootstrap": "1.4.0",
-    "react-dom": "18.2.0"
+    "react-dom": "^19.0.0",
+    "react-router-dom": "^7.1.1"
```

Comment:
Why is react-router-dom needed for React v19 upgrade? We use reach router.

Reply:
Yes, modified it.